<launch>

    <group ns="$(arg camera_name)">
        <node pkg="lirs_video_stream" type="video_stream" name="$(arg camera_name)_video_stream" output="screen">
            <param name="camera_name" type="string" value="$(arg camera_name)" />
            <param name="device_name" type="string" value="$(arg device_name)" />
            <param name="fps" type="int" value="$(arg fps)" />
            <param name="frame_id" type="string" value="$(arg frame_id)" />
            <param name="camera_info_url" type="string" value="$(arg camera_info_url)" />
            <param name="width" type="int" value="$(arg width)" />
            <param name="height" type="int" value="$(arg height)" />
            <param name="image_format" type="string" value="$(arg image_format)" />
        </node>

        <node if="$(arg show)" name="image_view" pkg="image_view" type="image_view">
            <remap from="image" to="image_raw" />
        </node>
    </group>

</launch>
